nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,83 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_allen_rsn_tmap_75_2011(data_dir=None, url=None, resume=True, verbose=1):
```

Comment:
>Would you call this an atlas? 

Group of RSNs can be called as an atlas.

>If you want to keep the atlas

That is how rest of the functions are kept as. I have no strong opinions on the names. My intention is just to keep them easy to call and consistent with other functionalities.